.github/instructions/codacy.instructions.md

```
@@ -7,9 +7,9 @@ Configuration for AI behavior when interacting with Codacy's MCP Server
 
 ## CRITICAL: After ANY successful `edit_file` or `reapply` operation
 - YOU MUST IMMEDIATELY run the `codacy_cli_analyze` tool from Codacy's MCP Server for each file that was edited, with:
-  - `rootPath`: set to the workspace path
-  - `file`: set to the path of the edited file
-  - `tool`: leave empty or unset
+ - `rootPath`: set to the workspace path
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)